web/packages/teleport/src/components/HeaderWithSteps/HeaderWithSteps.tsx

```diff
@@ -20,7 +20,7 @@ import Box from 'design/Box';
 import Text from 'design/Text';
 import React from 'react';
 
-export function DialogHeader({
+export function HeaderWithSteps({
```

Comment:
Since this component is meant to be used purely from within `StepSlider`, I think it could be a good idea to rename it to `StepHeader` and move it to `web/packages/teleport/src/components/StepSlider/StepHeader.tsx`.

If I didn't know the context and didn't work with `StepSlider` myself, it'd be hard for me to say what the purpose of `HeaderWithSteps` is.

Reply:
It may look like it's gonna be used just this way, but it's not. It's also used (indirectly) in the recovery flow, which is not a `StepSlider`. I'll address your comment by giving this component a documentation comment.

Reply:
On the other hand, compatibility with `StepSlider` is very deliberate here, so I think I'll follow your advice anyway. It shouldn't stop me from reusing this component outside the slider context.

Reply:
Adding some docs would work too, especially since I guess it's unlikely there would be more components added to `web/packages/teleport/src/components/StepSlider`.

Reply:
Yeah, I'll do both :)